components/script/dom/webglrenderingcontext.rs

```diff
@@ -584,15 +585,23 @@ impl Drop for WebGLRenderingContext {
 #[allow(unsafe_code)]
 unsafe fn typed_array_or_sequence_to_vec<T>(cx: *mut JSContext,
                                             sequence_or_abv: *mut JSObject,
-                                            config: <T as FromJSValConvertible>::Config) -> Result<Vec<T>, Error>
-    where T: ArrayBufferViewContents + FromJSValConvertible,
-          <T as FromJSValConvertible>::Config: Clone,
+                                            config: <T::Element as FromJSValConvertible>::Config)
+                                            -> Result<Vec<T::Element>, Error>
+    where T: TypedArrayElement,
+          T::Element: FromJSValConvertible + Clone,
+          <T::Element as FromJSValConvertible>::Config: Clone,
```

Comment:
You could have also done just `T::Element: FromJSValConvertible<Config = ()> + Clone` I think.